crypto/secp256k1/field.py

```diff
@@ -804,7 +1017,19 @@ def inverse(self):
         # This has a cost of 258 field squarings and 33 field multiplications.
         f = self
         fv = FieldVal
-        a2, a3, a4, a10, a11, a21, a42, a45, a63, a1019, a1023 = fv(), fv(), fv(), fv(), fv(), fv(), fv(), fv(), fv(), fv(), fv()
+        a2, a3, a4, a10, a11, a21, a42, a45, a63, a1019, a1023 = (
+            fv(),
+            fv(),
+            fv(),
+            fv(),
+            fv(),
+            fv(),
+            fv(),
+            fv(),
+            fv(),
+            fv(),
+            fv(),
```

Comment:
I'll write a function to fix this mess later.

Reply:
I missed that, I'll fix it and include it in the unformatted section below.